app/controllers/admin/payments_controller.rb

```diff
@@ -151,8 +151,13 @@ def create_invoice(csv, payments, payment_type, start_date, end_date = nil)
       csv << ["", Settings.mongoose_ledger_number, "Mongoose - #{ payment.member_id }", "9",
               payment.amount - Settings.mongoose_ideal_costs, ""]
     end
+    # minus lidmaatschap payments
+    activity_amount = payments.where(transaction_type: :activity).count - payments.where(
+      transaction_type: :activity,
+      transaction_id: Settings['intro.membership']
```

Comment:
Since the name of the activity is forced to "Lidmaatschap" anyway, consider filtering on that to somewhat resolve your todo

Maybe also put the "Lidmaatschap" in a variable somewhere instead of hardcoding it

Reply:
Don't know if i like the practice of filtering on a string that's so easiliy configured by an admin, the other consideration is you could also just change the activity id in settings